### PR TITLE
Restore php 5.6 through 7.0 compatibility

### PR DESCRIPTION
**Pull request by greg0ire**
## Changelog

```markdown
### Added
- php 5.6 - 7.0 compatibility
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ branches:
 language: php
 
 php:
+  - '5.6'
+  - '7.0'
```
Comment (jlamur) [inline .travis.yml]: Isn't this supposed to be managed by `dev-kit`?

Comment (greg0ire) [reply .travis.yml]: It very much is, and will be ignored / overwritten. Can't make this PR without ensuring it actually works, right?

Comment (jlamur) [reply .travis.yml]: Oh you'll create the PR on dev-kit after you have tested that no test fail, right?

Comment (greg0ire) [reply .travis.yml]: Yeah, not sure what the plan is regarding the merge order though :/

Comment (jlamur) [reply .travis.yml]: https://github.com/sonata-project/dev-kit/issues/315#issuecomment-339435597

**Pull request closed without merging**